src/components/chip-set/chip-set.tsx

```diff
@@ -62,6 +62,17 @@ export class ChipSet {
     @Prop({ reflect: true })
     public label: string;
 
+    /**
+     * Optional helper text to display below the chipset.
+     * When type is not `input`, the helper text is displayed below the
+     * input field when it has focus.
+     * When type is not `input`, the helper text is always displayed
```

Comment:
Both of these say "When type is not \`input\`" now 😄

I commented about saying something like "works just like for limel-input-field", but that was because I assumed we simply passed the helper-text value on to limel-input-field when type is `input`, but that doesn't seem to be the case?

Is the mdc-chip-set component handling the showing and hiding of the helper text, or are we doing that?

Reply:
> Both of these say "When type is not `input`" now 😄

Oh lord 🙈 my memory…

> Is the mdc-chip-set component handling the showing and hiding of the helper text, or are we doing that?

yes, for the type=input, it is MDC which is doing it. I didn't do anything special. Just added the helper line. For other types, I did some CSS rules for touchscreen and focus and hover.

Reply:
for input field, we say:
````tsx
    /**
     * Optional helper text to display below the input field when it has focus
     */
````

Reply:
Yeah, ok, then I think it's good as it is now. Nice work!

Reply:
Thanks :)

Reply:
Is it approved then? Should I squash and push?

Reply:
I know you saw the comment about whitespace. But yes, you can squash it. I'll approve when you've pushed the squashed commits.

Reply:
Ah OK. I'll remove the whitespace in the same time then :) I thought it was fine to leave it

Reply:
Done ✅